tools/status_monitor/README.md

```diff
@@ -0,0 +1,207 @@
+# Local OTA status monitor util
+
+A helper util for querying specific otaclient's status API and showing the OTA status in real time in the terminal window. If multiple ECUs' status are included, all of these ECUs' status(up to 10) will be displayed.
+
+## Usage
+
+This helper util requires otaclient installed. Prepare a virtual environment with otaclient installed, and then execute the package as follow:
+
+```bash
+# get the source code
+$ git clone https://github.com/tier4/ota-client
+...
+
+# prepare virtual environment
+$ python3 -m venv venv
+$ . venv/bin/activate
+(venv) $
+
+# install otaclient
+(venv) $ pip install -e ota-client
+...
+
+# under ota-client/ folder
+(venv) $ cd ota-client
+(venv) $ python3 -m tools.status_monitor --host <ecu_ip>
```

Comment:
Could you add how to exit this tool. I can exit from this tool by typing ctrl + c twice, but I'm not sure this is expected or not.

Reply:
This behavior is fixed at [fef3f12](https://github.com/tier4/ota-client/pull/251/commits/fef3f121f144a19cf2b750bdbb07130472da2f2d).